src/game/header/game.h

```diff
@@ -34,7 +34,7 @@
  * !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
  */
 
-#define GAME_API_VERSION 3
+#define GAME_API_VERSION 4
```

Comment:
Not the game api version, the savegame version somewhere in savegame.c. :) Or am I missunderstanding the change?

I thought: Bump the savegame version so that older client cannot load new savegames and add a simple fallback case so that newer clients can still load old savegames. There's already code for that, we've done it several times before.

Reply:
Sorry, looked to wrong place. Now should be better :-)